x-pack/plugins/viz_editor/pie_chart_plugin/pie_chart_plugin.tsx

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// @ts-ignore
+import { EuiSuperSelect } from '@elastic/eui';
+import { get, set } from 'lodash/fp';
+import React from 'react';
+import { columnSummary } from '../common/components/config_panel';
+import { IndexPatternPanel } from '../common/components/index_pattern_panel';
+import { Axis, selectColumn, ViewModel } from '../common/lib';
+import { EditorPlugin, PanelComponentProps } from '../public/editor_plugin_registry';
+
+interface PieChartPrivateState {
+  sliceAxis: Axis;
+  angleAxis: Axis;
+}
+
+type PieChartViewModel = ViewModel<'pieChart', PieChartPrivateState>;
+
+function dataPanel({ viewModel }: PanelComponentProps<PieChartViewModel>) {
+  return <IndexPatternPanel indexPatterns={viewModel.indexPatterns} />;
+}
```

Comment:
My other PR specifically changes the dataPanel function, please grab that code and update it here!